apps/ledger-live-mobile/src/components/SettingsRow.tsx

```diff
@@ -5,25 +5,33 @@ import { ChevronRightMedium, InfoMedium } from "@ledgerhq/native-ui/assets/icons
 import styled from "styled-components/native";
 import Touchable from "./Touchable";
 
-const StyledTouchableRow = styled(Touchable)<{
+const StyledTouchableRowContainer = styled(Flex)<{
```

Comment:
You can have called it juste like that as it's not touchable 
```suggestion
const StyledRowContainer = styled(Flex)<{
```

Just a minor comment can be ignored !